index.html

```diff
@@ -1,104 +1,134 @@
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                    </div>
                     <ul class="inner-flex-item">
                         <li class="pill-btn">Ruby on rails</li>
                         <li class="pill-btn">css</li>
                         <li class="pill-btn">javaScript</li>
                         <li class="pill-btn">html</li>
                     </ul>
-                <button class="flex-item4">See Project</button>
-            </div>
-        </article>
-        <article class="main-container">
-            <div class="grid-item"> 
-                <div class="flex-item1"><img src="img/Plaholder.png" alt="placeholder"></div>
-                <div class="flex-item2"><h2>Multi-Post Stories Gain+Glory</h2></div>
-                <ul class="inner-flex-item">
+                    <button class="flex-item4">See Project</button>
+                </div>
+            </article>
+            <article class="main-container">
+                <div class="grid-item">
+                    <div class="flex-item1"><img src="img/Plaholder.png" alt="placeholder"></div>
+                    <div class="flex-item2">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                    </div>
+                    <ul class="inner-flex-item">
                         <li class="pill-btn">Ruby on rails</li>
                         <li class="pill-btn">css</li>
                         <li class="pill-btn">javaScript</li>
                         <li class="pill-btn">html</li>
                     </ul>
-                <button class="flex-item4">See Project</button>
-            </div>
-        </article>
-        
-    </div> 
+                    <button class="flex-item4">See Project</button>
+                </div>
+            </article>
+
+        </div>
     </section>
-    
-        <section class="aboutme-container">
-            <div class="wrap">
-                <h2 class=" box-item headline"> About Me</h2>
-                <p class="box-item">  I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don’t hestiate to contact me.</p> 
-                <button class="fixx">Get My Resume</button>
-            </div>       
-        </section>
-    
+
+    <section class="form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form class="form-container" action="https://formspree.io/f/xjvlbpry" method="POST">
+
+            <input required class="form-item" maxlength="30" type="text" id="fullName" name="full_name"
+                placeholder="Full Name">
+
+            <input required class="form-item" type="email" id="mail" name="user_email" placeholder="Email Address">
+
+            <textarea required class="form-item" maxlength="500" id="msg" name="message_area">Enter test here</textarea>
```

Comment:
- [ ] Hi team, your project looks amazing!!! however, your `textarea` input should have `placeholder text` and not HTML text. This will allow users to write a message without having to delete the text you have there already before they could write a message. See suggestion below:

```suggestion
            <textarea required class="form-item" maxlength="500" id="msg" name="message_area" placeholder="Enter test here"></textarea>
```

Reply:
yes, I missed that.

Reply:
Done 👍